draft-ietf-quic-http.md

```diff
@@ -626,13 +644,39 @@ frames provide equivalent functionality. Frame type 0x9 is reserved.
 ### SETTINGS_ACK Frame {#frame-settings-ack}
 
 The SETTINGS_ACK frame (id = 0x0b) acknowledges receipt and application 
-of specific values in the peer's SETTINGS frame. It contains a list of 
-SETTINGS identifiers which the sender has understood and applied. This 
-list MAY be empty. 
+of specific values in the peer's SETTINGS frame. Depending on the stream where
+it is sent, it takes two different forms.
+
+On the connection control stream, it contains information about how and when the
+sender has processed the most recently-received SETTINGS frame, and has the
+following payload:
+
+~~~~~~~~~~~~~~~
+    0                   1                   2                   3
+    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                      Highest Local Stream (32)                |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                      Highest Remote Stream (32)               |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                    Recognized Identifiers (*)               ...
```

Comment:
The efficiency I'm looking for is in processing and understanding.  An endpoint that receives a negative list can more easily mark those settings as "do not try again".  As you say, a bit vector is the most compact, but given the relative frequency of settings changes, I'm not interested in optimizing that much.